rpm/update-spec-provides.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
```

Comment:
Probably want to add a `set -e` at least.  Otherwise a failure (for example) of `sed` will be ignored.